### PR TITLE
Pathfinder v1 Integration

### LINKED ISSUE #36: Increase vector bounding box size
Title is self explanatory, I found it a bit hard to hit in my experimentation

### LINKED ISSUE #63: Pathfinder integration
**Is your feature request related to a problem? Please describe.**  
No gaurentee that the path a user sees is the path that the robot will store.
**Describe the solution you'd like**  
When a drag is completed, pathfinder should be called to generate all the left and right wheel positions. Using all the robot information and dt. 

Once pathfinder completes computing paths the path on the screen should change so users can see the effects of their robot data on the curves.
**Describe alternatives you've considered**  
A dedicated preview/export button that shows the pathfinder curves.
**Additional context**  
Pathfinder should be an optional dependency and the program should detect if it is installed and only use this feature if available

### PR DESCRIPTION
**Pull request by kjrobrien**
Closes #63 and closes #36.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -82,6 +82,7 @@ dependencies {
     compile("si.uom", "si-units", "0.9")
     compile("systems.uom", "systems-common", "0.8")
     compile("com.google.code.gson", "gson", "2.8.5")
+    compile(files("lib/Pathfinder-Java.jar"))
```
Comment (SamCarlberg) [inline build.gradle.kts]: Is pathfinder not available as a maven dependency? @JacisNonsense 
And does this jar include the native libraries for all four supported platforms?

Comment (JacisNonsense) [reply build.gradle.kts]: It is on maven, but it isn't built for Linux 32/64, only ARM

Comment (kjrobrien) [reply build.gradle.kts]: I added the native build it was missing for Linux. I added a test that generates a path using Pathfinder and it has been running successfully on azure for all platforms.

**Comment by kjrobrien** [comment]
Here's a screenshot: 
image

**Pull request merged**